docs/acr42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,68 @@
+|                        | Understandability     | The queries should be understandable for ...?                                                                                         | Q:U4 |
+|                        | User error protection | ...?                                                                                                                                  | Q:U5 |
+| Performance efficiency | Time behaviour        | Response time of whole system? Response of database?                                                                                  | Q:P1 |
+|                        | Resource utilization  | How many do we have?                                                                                                                  | Q:P2 |
+|                        | Workload              | How many requests should be tolerable?                                                                                                | Q:P3 |
+|                        | Scalability           | How many sequences do we expect?                                                                                                      | Q:P4 |
+|                        |                       |                                                                                                                                       |      |
+| Maintainability        | Modularity            | Wie einfach sollen einzelne Komponenten (Web access, database, config) ausgetauscht werden können?                                    | Q:M1 |
+|                        | Reusability           | Welche Komponenten sollen auch getrennt benutzt werden können (database?)                                                             | Q:M2 |
+|                        | Analysability         | Wie effektiv und effizient soll das fertige Produkt analysiert werden können?                                                         | Q:M3 |
+|                        | Modifiability         | Welche Komponten sollen angepasst werden können? The (whole?) software should be easy to modify to cope with different virus species. | Q:M4 |
+|                        | Testability           | All single components should be tested. The whole system is tested from starting an instance to the user query.                       | Q:M5 |
+
+[//]: # (| Cultural and Regional? |)
+
+[//]: # (| Legal?)
+
+[//]: # (| Security?)
+
+[//]: # ()
+
+[//]: # (Scenarios:)
+
+## Stakeholders
+
+| Role                              | Expectations                                                           |
+|-----------------------------------|------------------------------------------------------------------------|
+| Database researcher (Alex)        | ?                                                                      |                                                                      
```

Comment:
Do we want to list individual people here? Who actually owns the system? Is it you as individual people or the department as a whole?